presto-accumulo/src/test/java/com/facebook/presto/accumulo/TestAccumuloDistributedQueries.java

```diff
@@ -0,0 +1,503 @@
+//                "SELECT 1");
+
+        this.assertCreateTableAsSelect(
+                "test_with_data",
+                "SELECT * FROM orders WITH DATA",
+                "SELECT * FROM orders",
+                "SELECT count(*) FROM orders");
+
+        this.assertCreateTableAsSelect(
+                "test_with_no_data",
+                "SELECT * FROM orders WITH NO DATA",
+                "SELECT * FROM orders LIMIT 0",
+                "SELECT 0");
+
+//        No sample tables
+//        this.assertCreateTableAsSelect(
+//                "test_sampled",
+//                "SELECT orderkey FROM tpch_sampled.tiny.orders ORDER BY orderkey LIMIT 10",
+//                "SELECT orderkey FROM orders ORDER BY orderkey LIMIT 10",
+//                "SELECT 10");
+
+        // Tests for CREATE TABLE with UNION ALL: exercises PushTableWriteThroughUnion optimizer
+
+//        Function "UUID" not found
+//        this.assertCreateTableAsSelect(
+//                "test_union_all",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders WHERE orderkey % 2 = 0 UNION ALL " +
+//                        "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders WHERE orderkey % 2 = 1",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders",
+//                "SELECT count(*) FROM orders");
+
+//        Function "UUID" not found
+//        this.assertCreateTableAsSelect(
+//                getSession().withSystemProperty("redistribute_writes", "true"),
+//                "test_union_all",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders UNION ALL " +
+//                        "SELECT UUID() AS uuid, DATE '2000-01-01', 1234567890, 1.23",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders UNION ALL " +
+//                        "SELECT UUID() AS uuid, DATE '2000-01-01', 1234567890, 1.23",
+//                "SELECT count(*) + 1 FROM orders");
+
+//        Function "UUID" not found
+//        this.assertCreateTableAsSelect(
+//                getSession().withSystemProperty("redistribute_writes", "false"),
+//                "test_union_all",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders UNION ALL " +
+//                        "SELECT UUID() AS uuid, DATE '2000-01-01', 1234567890, 1.23",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders UNION ALL " +
+//                        "SELECT UUID() AS uuid, DATE '2000-01-01', 1234567890, 1.23",
+//                "SELECT count(*) + 1 FROM orders");
+    }
+
+    @Override
+    public void testDelete()
+            throws Exception
+    {
+        try {
+            // TODO Deletes are not supported by the connector
+            super.testDelete();
+        }
+        catch (Exception e) {
+            assertUpdate("DROP TABLE test_delete");
```

Comment:
Not sure what I was thinking here.  This should just be overridden with a comment like `testAddColumn` will be.